javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,37 @@
+console.log('Script loaded');
+let products = [];
+
+const testProductNames = ['Flat screen', 'Mobile phone', 'Wallet'];
+
+function renderProducts() {
+    const productsAvailable = getAvailableProducts();
+    const ulTag = document.querySelector("section > ul");
+
+    for (let i = 0; i < productsAvailable.length; i++) {
+        const liTag = document.createElement('li');
+        const childUlTag = document.createElement("ul");
+        for (let j = 1; j <= 4; j++) {
+            const childLiTag = document.createElement("li");
+            childLiTag.setAttribute('class', Object.keys(productsAvailable[i])[j]);
+            childLiTag.innerHTML = Object.values(productsAvailable[i])[j];
+            childUlTag.appendChild(childLiTag);
+        }
+        liTag.appendChild(childUlTag);
+        ulTag.appendChild(liTag);
+    }
+}
+function getAvailableProducts() {
```

Comment:
this function isn't needed `getAvailableProducts` is provided in the `hyfBayHelpers.js` file